services/datamanager/builtin/file_deletion.go

```diff
@@ -0,0 +1,111 @@
+package builtin
+
+import (
+	"context"
+	"errors"
+	"io/fs"
+	"math"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"go.viam.com/rdk/logging"
+
+	"github.com/ricochet2200/go-disk-usage/du"
+	"go.viam.com/rdk/services/datamanager/datacapture"
+	"go.viam.com/rdk/services/datamanager/datasync"
+)
+
+// TODO change these values back to what they should be
+const (
+	fileDeletionThreshold    = .2
+	captureDirRatioThreshold = .1
+	n                        = 4
+)
+
+var errAtSizeThreshold = errors.New("capture dir is at correct size")
+
+func checkFileSystemStats(ctx context.Context, captureDirPath string, logger logging.Logger) (bool, error) {
```

Comment:
Perhaps rename to something like `shouldDeleteBasedOnDiskUsage` to describe what's being returned and why.